documentation/dev-portal/src/events/cryptotalk/nym-vpn-faq.md

```diff
@@ -22,7 +22,7 @@ client ───► Gateway ──┘  mix  │  mix  ┌─►mix ───►
                          mix  └─►mix──┘  mix
 ```
 
-The 2-hop option is going from the local client -> Entry Gateway -> directly to Exit Gateway -> internet. This option is good for operations demanding faster connection. Keep in mind that this setup by-passes the 3 layers of Mix Nodes. 
+The 2-hop option is going from the local client -> Entry Gateway -> directly to Exit Gateway -> internet. This option is good for operations demanding faster connection. Keep in mind that this setup by-passes the 3 layers of Mix Nodes, however the anonymising features happening in your local client still apply.
```

Comment:
"however the anonymising features happening in your local client still apply."

This sentence is a little vague - maybe spell these out a bit?